NEWS_abbr.md

```diff
@@ -0,0 +1,287 @@
+others are due to modifications of function names, argument names, or argument
+positions. There are too many to list here, so you should refer to the [Python
+Polars API docs](https://docs.pola.rs/api/python/dev/reference/index.html).
+
+Installation instructions for the rewritten version are now identical to the
+instructions for the old version. The old version (referred to as `polars0` in
+the docs) can be installed from R-universe with:
+
+```r
+Sys.setenv(NOT_CRAN = "true")
+install.packages("polars0", repos = "https://rpolars.r-universe.dev")
+```
+
+### Breaking changes
+
+* The class names of `polars` objects have changed:
+  - `RPolarsLazyFrame` -> `polars_lazy_frame`
+  - `RPolarsDataFrame` -> `polars_data_frame`
+  - `RPolarsSeries` -> `polars_series`
+  - `RPolarsExpr` -> `polars_expr`
+
+* Conversion from unknown classes to Polars objects now fail. Developers can
+  specify how those objects should be handled by `polars` by creating a method
+  for `as_polars_series.<my-object>`.
+
+  ```r
+  ### OLD
+  a <- 1
+  class(a) <- "foo"
+  as_polars_series(a)
+  #> polars Series: shape: (1,)
+  #> Series: '' [f64]
+  #> [
+  #>         1.0
+  #> ]
+  ```
+
+  ```r
+  ### NEW
+  a <- 1
+  class(a) <- "foo"
+  as_polars_series(a)
+  #> Error:
+  #> a <foo> object can't be converted to a polars Series.
+  #> Run `rlang::last_trace()` to see where the error occurred.
+  ```
+
+* Conversion from `polars` to R types has been revamped: `$to_r()`, `$to_list()`
+  and `$to_data_frame()` no longer exist. Instead, you must use `as.data.frame()`,
+  `as.list()` and `as.vector()`.
+
+  If finer control is needed (for instance to specify how integer-64 or date
+  columns should be handled), use either `$to_r_vector()` or set options for the
+  entire session with `polars_options()`.
+
+* In general, `polars` now uses dots (`...`) in two scenarios:
+
+  1. to pass an unlimited number of inputs (for instance in `select()`, `cast()`,
+     or `group_by()`), using [dynamic-dots](https://rlang.r-lib.org/reference/dyn-dots.html).
+
+    For example, if you used to pass a vector of column names or a list of
+    expressions, you now need to expand it with `!!!`:
+    ``` r
+    ### OLD
+    dat <- as_polars_df(head(mtcars, 3))
+    my_exprs <- list(pl$col("drat") + 1, "mpg", "cyl")
+    dat$select(my_exprs)
+    #> shape: (6, 3)
+    #> ┌──────┬──────┬─────┐
+    #> │ drat ┆ mpg  ┆ cyl │
+    #> │ ---  ┆ ---  ┆ --- │
+    #> │ f64  ┆ f64  ┆ f64 │
+    #> ╞══════╪══════╪═════╡
+    #> │ 4.9  ┆ 21.0 ┆ 6.0 │
+    #> │ 4.9  ┆ 21.0 ┆ 6.0 │
+    #> │ 4.85 ┆ 22.8 ┆ 4.0 │
+    #> └──────┴──────┴─────┘
+
+    ### NEW
+    dat$select(!!!my_exprs)
+    #> shape: (3, 3)
+    #> ┌──────┬──────┬─────┐
+    #> │ drat ┆ mpg  ┆ cyl │
+    #> │ ---  ┆ ---  ┆ --- │
+    #> │ f64  ┆ f64  ┆ f64 │
+    #> ╞══════╪══════╪═════╡
+    #> │ 4.9  ┆ 21.0 ┆ 6.0 │
+    #> │ 4.9  ┆ 21.0 ┆ 6.0 │
+    #> │ 4.85 ┆ 22.8 ┆ 4.0 │
+    #> └──────┴──────┴─────┘
+    ```
```

Comment:
Could you split code blocks old and new?

Reply:
[5ea8298](https://github.com/eitsupi/neo-r-polars/pull/284/commits/5ea829822d6ab2d818e403bb17a582a313a09446)